geth/proxy/transactions/sendtransaction.go

```diff
@@ -0,0 +1,299 @@
+package transactions
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"math/big"
+
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/les/status"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/robertkrimen/otto"
+	"github.com/status-im/status-go/geth/common"
+)
+
+// contains sets of possible defines the name for a giving transaction.
+const (
+	SendTransactionName = "eth_sendTransaction"
+)
+
+// ExecuteRemoteSendTransaction defines a funct
+func ExecuteRemoteSendTransaction(manager common.RPCNodeManager, req common.RPCCall, call otto.FunctionCall) (*otto.Object, error) {
+	resp, _ := call.Otto.Object(`({"jsonrpc":"2.0"})`)
+	resp.Set("id", req.ID)
+
+	selectedAcct, err := manager.Account().SelectedAccount()
+	if err != nil {
+		return nil, err
+	}
+
+	toAddr := req.ParseFromAddress()
+	gas := (*big.Int)(req.ParseGas())
+	dataVal := []byte(req.ParseData())
+	priceVal := (*big.Int)(req.ParseValue())
+	gasPrice := (*big.Int)(req.ParseGasPrice())
+
+	tx := types.NewTransaction(uint64(req.ID), toAddr, priceVal, gas, gasPrice, dataVal)
+	txs, err := types.SignTx(tx, types.HomesteadSigner{}, selectedAcct.AccountKey.PrivateKey)
```

Comment:
`v` is one of the three signature parameters: `v`, `r`, `s`. `v` internally must be set to `chainID` before signing the transaction. `EIP155Signer` is the signer which does it, computes the signature and replaces `v`, `r`, `s` with the real signature.